### PR TITLE
shellcheck: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/shellcheck.md [hunk 1]
```diff
@@ -8,26 +8,30 @@
...
+- Enable one or more [o]ptional checks:
 
-`shellcheck --enable={{add-default-case|avoid-nullary-conditions}}`
+`shellcheck --enable {{add-default-case,avoid-nullary-conditions,...}} {{path/to/script.sh}}`
 
 - List all available optional checks that are disabled by default:
 
 `shellcheck --list-optional`
+
+- Adjust the level of [S]everity to consider (default severity is `style`):
```
Comment (acuteenvy) [inline pages/common/shellcheck.md:35]: For consistency with other examples
> Display output in the specified [f]ormat (defaults to `tty`):
```suggestion
- Adjust the level of [S]everity to consider (defaults to `style`):
```

**Review by acuteenvy — APPROVED** [review]

File: pages/common/shellcheck.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
 # shellcheck
 
-> Shell script static analysis tool.
-> Check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
+> Statically check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
```
Comment (vitorhcl) [inline pages/common/shellcheck.md:3]: Let me take this opportunity to adequate the description to our style, what do you guys think of this?

File: pages/common/shellcheck.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
 
-- Check a shell script interpreting it as the specified shell dialect (overrides the shebang at the top of the script):
+- Check a shell script interpreting it as the specified [s]hell dialect (overrides the shebang at the top of the script):
 
 `shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/script.sh}}`
 
-- Ignore one or more error types:
+- Ignor[e] one or more error types:
 
 `shellcheck --exclude {{SC1009,SC1073}} {{path/to/script.sh}}`
```
Comment (vitorhcl) [inline pages/common/shellcheck.md:16]:
```suggestion
`shellcheck --exclude {{SC1009,SC1073,...}} {{path/to/script.sh}}`
```
For consistency with the page

File: pages/common/shellcheck.md [hunk 4]
```diff
@@ -1,33 +1,36 @@
 # shellcheck
 
-> Shell script static analysis tool.
-> Check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
+> Statically check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
 > More information: <https://www.shellcheck.net>.
```
Comment (vitorhcl) [inline pages/common/shellcheck.md:4]:
```suggestion
> More information: <https://github.com/koalaman/shellcheck/wiki>.
```
We can point to the GitHub Wiki instead

**Review by vitorhcl — APPROVED** [review]
LGTM after my suggestions

**Pull request merged**